lib/standard/kernel.nit

```diff
@@ -660,6 +695,120 @@ universal Byte
 	redef fun to_i is intern
 	redef fun to_f is intern
 	redef fun to_b do return self
+	redef fun to_i8 is intern
+	redef fun to_i16 is intern
+	redef fun to_u16 is intern
+	redef fun to_i32 is intern
+	redef fun to_u32 is intern
+
+	redef fun distance(i) do return (self - i).to_i
+
+	redef fun <=>(other)
```

Comment:
In Byte, <=> and other methods of Comparable/Distrete seems useless since the inherited implementation is sane. Moreover, it is unrelated to the current PR.